src/js/store/index.js

```diff
@@ -0,0 +1,40 @@
+    const defaultMenuCategory = {
+        espresso : [],
+        frappuccino: [],
+        blended: [],
+        teavana: [],
+        desert: []
+    }
+
+export function fetchMenu() {
+    let menuItem = JSON.parse(localStorage.getItem("menu"));
+    if(!menuItem) {
+        menuItem = defaultMenuCategory
+        localStorage.menu = JSON.stringify(menuItem);
+    };
```

Comment:
```suggestion
    let menuItem = JSON.parse(localStorage.getItem("menu")) || defaultMenuCategory;
    localStorage.menu = JSON.stringify(menuItem);
```
이렇게 변경해도 같은 동작을 할 것 같아요!